### ISSUE TITLE
continue events after logout

### ISSUE THREAD

**Issue by ZengineAlex** [ZengineAlex/game#12]
events like upgrade a building or research technologies must continue after logout

**Comment by ZengineChris** [comment]
This can be calculated when the user logged in again. If the user don't get a mail if something happen, but also this can a simple cron job handle.

**Comment by ZengineAlex** [comment]
yes i think a scheduled task via laravel could work very well. the task has to be executed at the time the upgrade or research is finished.
